docs/03_server/05_access-control/03_password_authentication.md

```diff
@@ -530,51 +631,84 @@ Note the following:
+
+## Full permissions list
+
+Most of the message workflows described in this page are permissioned based on a set of default rights which are provided in the form of a CSV file as part of the platform authentication distribution. Most permissions and their relationship to events have been explained in their own sections, and the whole table can be found below for reference:
+
+| Code           | Description                    |
+|----------------|--------------------------------|
+| INSERT_PROFILE | Add a new Profile              |
+| INSERT_USER    | Add a new User                 |
+| AMEND_PROFILE  | Amend an existing Profile      |
+| AMEND_USER     | Amend an existing User         |
+| CHANGE_PWD     | Change another User's password |
+| DELETE_PROFILE | Delete a Profile               |
+| DELETE_USER    | Delete a User                  |
+| DISABLE_USER   | Disable a User                 |
+| ENABLE_USER    | Enable a User                  |
+| EXPIRE_PWD     | Expire another User's password |
+
+A default Profile record is also provided named USER_ADMIN which contains all the previous rights.
+
+:::info
+Users can change their own password as well as expire their own password (assuming they are logged in). However, it is only possible to change/expire another user's password if you have administrator rights.
+:::
+
+## Event auditing
+
+Each one of these events is audited in one way or another, either by using the automatic mechanism provided at the table definition level (e.g. PROFILE, PROFILE_RIGHT, PROFILE_USER, PASSWORD_RESET, USER and USER_ATTRIBUTES), or by providing custom tables with the audit information.
+
+In the first case scenario, auditing works as it would do for any other genesis table: AUDIT_EVENT_TYPE reflects the event message type (i.e. EVENT_INSERT_USER), AUDIT_EVENT_TEXT may contain a free text field provided by the user calling the event, AUDIT_EVENT_DATETIME is autogenerated with the current date and time the change happened and AUDIT_EVENT_USER corresponds to the user who triggered the event in question.
+
+In the second case scenario, we have automatic events to log changes in USER_AUDIT and USER_ATTRIBUTES when a password expires. And we also have specific handling for USER_LOGIN audits. The USER_LOGIN_AUDIT table will contain entries for the following events: LOGIN, LOGOUT, SESSION_EXPIRED, REJECTED (only available if a maximum number of user sessions has been configured), FAILED_LOGIN, and FAILED_LOGOUT (if an incorrect session ID or user name has been provided). 
+
+Additionally, all USER_LOGIN audit events are logged to a file at INFO level in the following format:
+
+    25 Jan 2024 16:31:18.823 12729 [dbCoroutinesContext-4 @coroutine#994] INFO  global.genesis.auth.manager.controller.LoginAuditController - AuditLoginEvent: UserLoginAudit{serialVersionUID='1',userLoginAuditId={not-set}, userName=JohnDoe, authAction=LOGIN, ipAddress=/127.0.0.1, reason=, recordId={not-set}, timestamp={not-set}}
```

Comment:
Also adding a sample line of what a log statement would look like after a successful login.